tests/integration_tests/migrations/87d38ad83218_migrate_can_view_and_drill_permission__tests.py

```diff
@@ -34,28 +36,28 @@
 downgrade = migration_module.do_downgrade
 
 
+@pytest.mark.usefixtures("app_context")
```

Comment:
@betodealmeida I tried to standardize this. It _seems_ that:

1. If you want to use a fixture in a test you can either pass it as an argument or via [@pytest.mark.usefixtures](https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#use-fixtures-in-classes-and-modules-with-usefixtures) if test functions do not directly need access to a fixture object. Personally I find the later more readable as otherwise when you're reading through the code it feels like a potentially unused argument.
2. Somewhat cryptic, but if you want to use a fixture in another fixture you have to pass it as an argument*. 

* Per [here](https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#use-fixtures-in-classes-and-modules-with-usefixtures) it states, 

> Warning: Note this mark has no effect in fixture functions. For example, this will not work as expected:
> 
> ```python
> @pytest.mark.usefixtures("my_other_fixture")
> @pytest.fixture
> def my_fixture_that_sadly_wont_use_my_other_fixture():
>    ...
> ```
>
> Currently this will not generate any error or warning, but this is intended to be handled by [issue #3664](https://github.com/pytest-dev/pytest/issues/3664).


Maybe sticking with one pattern is cleaner though. I'm open to suggestions.

Reply:
I spoke with @michael-s-molina offline and it seems like he's supportive of the approach I've taken here so I'm going to go ahead and merge this. I think as a follow up it would be good to use the `@pytest.mark.usefixtures` decorator where possible.